docs/architecture/grpc-for-wcf-developers/protobuf-repeated.md

```diff
@@ -15,9 +15,11 @@ message Person {
 }
 ```
 
-In the generated code, `repeated` fields are represented by the `Google.Protobuf.Collections.RepeatedField<T>` generic type rather than any of the built-in .NET collection types.
+In the generated code, `repeated` fields are represented by read-only properties of the [`Google.Protobuf.Collections.RepeatedField<T>`][repeated-field] type rather than any of the built-in .NET collection types. This type implements all the standard .NET collection interfaces, such as <xref:System.Collections.Generic.IList%601> and <xref:System.Collections.Generic.IEnumerable%601>. So you can use LINQ queries or convert it to an array or a list easily.
 
-The `RepeatedField<T>` type includes the code required to serialize and deserialize the list to the binary wire format. It implements all the standard .NET collection interfaces, such as <xref:System.Collections.Generic.IList%601> and <xref:System.Collections.Generic.IEnumerable%601>. So you can use LINQ queries or convert it to an array or a list easily.
+The `RepeatedField<T>` type includes the code required to serialize and deserialize the list to the binary wire format.
+
+[repeated-field]: https://developers.google.cn/protocol-buffers/docs/reference/csharp/class/google/protobuf/collections/repeated-field-t-
```

Comment:
Not sure what syntax this is, but it doesn't render- https://review.docs.microsoft.com/en-us/dotnet/architecture/grpc-for-wcf-developers/protobuf-repeated?branch=pr-en-us-19908

Reply:
That's "reference link syntax" ([Markdown spec](https://daringfireball.net/projects/markdown/syntax#link), [CommonMark spec](https://spec.commonmark.org/0.29/#reference-link)). It lets the destination be somewhere other than right after the linked text. It looks like it's rendering fine to me: the `Google.Protobuf.Collections.RepeatedField<T>` in the sentence that start "In the generated code, ..." is hyperlinked in the rendered output. Screenshot below.

I find it more readable for long links, and it's very useful if the same destination needs to be used multiple places.

<hr>

![screenshot of hyperlinked RepeatedField](https://user-images.githubusercontent.com/10605667/91128187-92a6b500-e65c-11ea-8af3-6f22519113af.png)

Reply:
Apologies. I stand corrected